frontend/src/app/features/user-preferences/notifications-settings/table/notification-settings-table.component.html

```diff
@@ -3,18 +3,18 @@
     <table class="generic-table">
       <colgroup>
         <col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
-        <col highlight-col>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
+        <col opHighlightCol>
       </colgroup>
-      <thead>
+      <thead class="-sticky">
```

Comment:
Why did you add the `-sticky` class here? If I see that correct, it can't have any effect as the table itself isn't scrollable.

Reply:
Yeah, for this bug it doesn't do anything. I added this class here to be the same as other tables like projects and workpackages tables.

Reply:
For the workPackages and projects table it makes sense because these tables have an inner scrolling meaning that you can scroll the rows and the rest of the page remains static. In that case it makes sense to make the header sticky. Without that, it has no effect in the best case or unwanted site effects in the worst case. So please remove it from here. 
Afterwards it is good to merge 👍

Reply:
Ah yeah I got it, thanks for explanation.